obinlog/proto/binlog.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto2";
+package obinlog;
+import "gogoproto/gogo.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+// for date and time type, string is set
+
+// for int type int64 or uint64 is set
+// for float, double, double is set
+// for decimal, string is set in human-readable now
+// for bit, bytes is set
+
+// for text and char type, string is set
+// for blob and binary type, bytes is set
+// for enum, set, uint64 is set
+// for json, bytes is set
+message Column {
+    optional bool is_null = 1 [default = false];
+
+    optional int64 int64_value = 2;
+    optional uint64 uint64_value = 3;
+    optional double double_value = 4;
+    optional bytes bytes_value = 6;
```

Comment:
5? check and fix similar problems

Reply:
i think it's ok with this,  no need to use continuous number and should not change, but i can change it now as no servise use this proto now

Reply:
I know it's ok,  but ulgy